storage/src/journal/variable.rs

```diff
@@ -332,22 +305,10 @@ impl<E: Storage + Metrics> Journal<E> {
     /// The `concurrency` parameter controls how many blobs are replayed concurrently. This can dramatically
     /// speed up the replay process if the underlying storage supports concurrent reads across different
     /// blobs.
-    ///
-    /// # Prefix
```

Comment:
`archive` used to use to "in theory" lower IO on restart. By not doing it, however, we can perform a quick checksum over disk before even starting.